MonkeyChat/ChatViewController.swift

```diff
@@ -72,13 +72,17 @@ class ChatViewController: JSQMessagesViewController, JSQMessagesComposerTextView
   var descriptionView = UIView()
   var nameLabel = UILabel()
   var statusLabel = UILabel()
+  var descriptionViewTapRecognizer: UITapGestureRecognizer!
   
   // VIEW - navigation bar - right button
   var negativeSpacerBarButtonItem = UIBarButtonItem(barButtonSystemItem: UIBarButtonSystemItem.fixedSpace, target: self, action: nil)
   var avatarBarButtonItem = UIBarButtonItem()
   var avatarButton = UIButton(type: UIButtonType.custom)
   var avatarImageView = UIImageView()
   
+  // VIEW - messages - audio bubble
+  var audioBubbleOnPlay: RGCircularSlider!
```

Comment:
This var won't always be different than `nil`, so let's put this as an optional `?`